examples/gcp_defaults/main.tf

```diff
@@ -0,0 +1,27 @@
+provider "vault" {
+  address = "http://localhost:8200"
+  token   = "root"
+}
+
+variable "project" {
+  default = "test"
```

Comment:
Consider adding a description and type to this

Reply:
good spot - fixed